File: C/Assignment9_2.c
//////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//
// file name   : Assignment9_2.c 
// Description : Write a program which accept number from user and whether it is contain 0 in it or not .
// Author      : Aniket Devaba Markad 
// Date        : 16/05/2025
// 
////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////// 

#include<stdio.h>
#define TRUE 1
#define FALSE 0
typedef int BOOL;

////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////// 
// 
// Function Name : ChkZero
// Description   : This function is used to check whether it is contain 0 in it or not. 
// Input         : int 
// Output        : void
// Author        : Aniket Devaba Markad 
// Date          : 16/05/2025 
// 
///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////// 


BOOL ChkZero(int iNo)
{
    int iDigit = 0;
    if(iNo<0)
    {
        iNo=-iNo;
    }
    while (iNo!=0)
    {
        iDigit=iNo%10;
        if(iDigit==0)
        {
            return TRUE;
        }
        iNo = iNo / 10;
    }
    return FALSE;
}
int main()
{
    int iValue =0;
    BOOL bRet = FALSE;
    printf("enter number");
    scanf("%d",&iValue);

    bRet= ChkZero(iValue);
    if(bRet == TRUE)
    {
        printf("it contains zero");
    }
    else{
        printf("There is no zero");
    }
    return 0;
}

File: Java/Assignment9_2.java
/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//
// file name   : Assignment9_2.java
// Description : write a program which Accept number from user and check whether the zero is present or not
// input.      : 2034
// output.     : 4
//               3
//               0.  present
//               2. 
// Author      : Aniket Devaba Markad                        
// Date        : 1/1/2026
//
////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////

import java.util.Scanner;

class  Digit
{

/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//
// Function Name : CheckDigit
// Description   : This function is used to check whether the number is present or not 
// Author        : Aniket Devaba Markad 
// Date          : 1/1/2026 
// 
///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////

    boolean DisplayDigit(int iNo)
    {
        boolean bFlage = false;
        int iDigit = 0;
        if(iNo < 0)
        {
            iNo = -iNo;
        }
       
        while(iNo != 0)
        {

        iDigit = iNo % 10;
        if(iDigit == 0)
        {
            bFlage = true;
            break;  
        }
        iNo = iNo/10;
        }

        return bFlage;
    }
   
}
public class Assignment9_2
{
    public static void main(String a[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("enter the number  ");
        int iValue = sobj.nextInt();

        Digit dobj = new Digit();
        boolean bRet = dobj.DisplayDigit(iValue);

        if(bRet == true)
        {
            System.out.println("zero is present ");
        }
        else
        {
            System.out.println("zero is not present");
        }
    }
}








